src/components/Guilds/Filter/index.tsx

```diff
@@ -24,15 +26,37 @@ const FilterRow = styled.div`
   }
 `;
 
+const ButtonContainer = styled(Flex)`
+  flex-direction: Row;
+`;
+
 export const Filter = () => {
   const [viewFilter, setViewFilter] = useState(false);
+  const [createProposal, setCreateProposal] = useState(true);
   const { totalFilters } = useFilter();
+
+  const history = useHistory();
+  const location = useLocation();
+
   return (
     <FilterContainer>
       <FilterRow>
         <InputText placeholder="Proposal title" />
-        {isDesktop && <FilterMenu />}
-        {isMobile && (
+        {createProposal && (
+          <ButtonContainer>
+            <Button onClick={() => setCreateProposal(false)}>

```

Comment:
Im confused, what is this button and `createProposal` value doing?

Reply:
the design's changed but I did not want to remove the previous dropdown menu's completely so this would be a stop gap.